### ISSUE TITLE
Allow disabling any and all automatic scrolling or collapsing of on-screen content

### ISSUE THREAD

**Issue by burnpanck** [jupyter/notebook#7317]
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository: *Maybe, maybe not. For the best I try, I seem to have 50 % odds at hitting the right repo among notebook/jupyterlab.*

### Problem

When working interactively, screen jerk is major UX pain. If you let a long-running notebook run in the background, it may indeed be beneficial if it directs you to some well-thought point of interest when you come back to the window, but while I am actively working with a notebook, any movement on screen that I did not actively command is just disorienting. Of course, if output is being  generated within the view that I see, then some parts will have to move, and it's a design question which parts should stay still. But as long as input cells or append-only output is on screen, there is at least one reasonable choice of what part should not move.

In the last year or so (somewhere between the move to the jupyterlab backend / notebook v7 and now), I feel that this part of UX has considerably deteriorated. In fact, there are so many issues related to those issues in the two repos, combined with a wealth of outdated information related to pre v7 notebook, that I find it hard to figure out where and what to look and/or report.

From my perspective, it feels like there are some half-baked features in current versions of notebook that seriously hurt user-experience, and I wish there was a way to just dumb it down / disable all of that.

### Proposed Solution

 * Identify and document what the set of intended features related to scroll and collapse are
 * Provide an option to disable any and all scroll - this would allow me to clearly open a bug-report against that feature whenever I encounter UI jerk.

**Comment by burnpanck** [comment]
> Are you looking for all outputs to scroll when they reach a certain length?

No. Indeed, the behaviour of automatic output collapse in jupyter notebook 7 is *also* worse than before, I filed issues in that regard too: https://github.com/jupyter/notebook/issues/7003. However, here, it's purely about the scroll behaviour of the overall notebook content (that is, cells in the browser, not output in the cell output area). I would rather not switch my workflow to Visual Studio Code, just because of erratic scroll in the browser. Again: The desired behaviour is very simple: The notebook should not scroll on it's own, period. It appears to me that jupyter lab / jupyter notebook introduced new behaviours intended to provide a better UX, but IMHO, it's become worse. Because these are clearly "features" (it's not a browser default behaviour, but something explicitly implemented in jupyter), I assume it is conceivable to disable them.

You are right that I did log an issue about one particularly annoying case of erratic scroll (https://github.com/jupyterlab/jupyterlab/issues/15968), in that case it is clearly a bug. However, I also noticed for example that the notebook automatically scrolls when I enter a command in the python debugger (`%debug` magic) in the jupyter notebook. The behaviour I observed there may arguably be considered "as intended" - I still find it disorienting. Rather than trying to start a discussion about every single case of intended vs unintended and useful vs. annoying automatic scroll, this specific feature request here is to indicate that some people find screen jerk disorienting  while others may potentially find it useful. A configuration option that does not address specific features, but addresses the screen jerk would cater to us.

**Comment by zvorgiygeonka** [comment]
Please solve this issue, finally. For several years, I have not updated Jupyter Lab because there is no option to turn off this scrolling, which annoys me. I usually do not participate in GitHub discussions, but this issue really annoys me; just solve it, finally, please.
